src/components/CodeView/index.tsx

```diff
@@ -197,12 +206,17 @@ export class CodeViewBase extends React.Component<Props> {
                               {enableCommenting && addCommentButton}
                             </td>
 
-                            <td className={styles.code}>
-                              {renderCode({
-                                code,
-                                language,
-                              })}
-                            </td>
+                            <Profiler
+                              id="CodeView-Highlighting"
+                              onRender={this.onRenderCallback}
+                            >
```

Comment:
So we are going to profile every single line of code?

How are we going to determine that the `Profiler` isn't slowing down the page more than without `Profiler`? I read that impact should be minimal here: https://gist.github.com/bvaughn/25e6233aeb1b4f0cdb8d8366e54a3977#gistcomment-2744384, but this part of our code is going to render a `Profiler` for every single LOC.

Reply:
Ah, I didn't realize we were highlighting the code one line at a time - I thought we were doing it to all the code at once. It's odd, because when I tested this locally, I only saw one call to send the data to GA for highlighting per render, not a call for every line of code, so I wonder what exactly is happening here? Maybe `onRender` is only getting called once? If so, what exactly is being measured?

It's fine to just remove this for now, as I think we're more concerned at the moment with measuring overall performance of the component and not trying to isolate highlighting performance. That will resolve this particular issue.